build.sbt

```diff
@@ -327,13 +316,10 @@ lazy val compileInterfaceProj = (project in compilePath / "interface").
     // needed because we fork tests and tests are ran in parallel so we have multiple Scala
     // compiler instances that are memory hungry
     javaOptions in Test += "-Xmx1G",
-    artifact in (Compile, packageSrc) := Artifact(srcID).copy(configurations = Compile :: Nil).extra("e:component" -> srcID)
+    publishArtifact in (Compile, packageSrc) := true,
+    publishMavenStyle := true
```

Comment:
I don't know if it is necessary, but the idea was to be able to publish the compiler interface in a most maven-friendly way, no?